Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -106,6 +109,7 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
             payLaterToggle,
             newPayPalCheckoutToggle,
             contactInformationToggle,
+            amountBreakdownToggle,
```

Comment:
Are you able to share a screenshot of this as well? Just want to make sure the toggles aren't getting too squished.